enclave-tls/src/verifiers/sgx-la/verify_evidence.c

```diff
@@ -4,17 +4,17 @@
  */
 
 #include <enclave-tls/log.h>
-#include <enclave-tls/enclave_quote.h>
+#include <enclave-tls/verifier.h>
 #include "sgx_error.h"
 #include "sgx_la.h"
 #include "sgx_stub_u.h"
 #include "sgx_quote_3.h"
 #include "sgx_dcap_ql_wrapper.h"
 
 /* Refer to explanation in sgx_la_collect_evidence */
-enclave_quote_err_t sgx_la_verify_evidence(enclave_quote_ctx_t *ctx,
-					   attestation_evidence_t *evidence, uint8_t *hash,
-					   unsigned int hash_len)
+enclave_verifier_err_t sgx_la_verify_evidence(enclave_verifier_ctx_t *ctx,
+					      attestation_evidence_t *evidence, uint8_t *hash,
+					      uint32_t hash_len)
```

Comment:
why change to uint32_t?

Reply:
unify it wiht uint8_t

Reply:
I mean using size_t is better. We don't need to care about the length of hash_len variable.

Reply:
size_t is explained on different hardware architecutre, but uint32_t just has one voice on different HW ARCH.